homeassistant/components/unifi/entity.py

```diff
@@ -103,6 +103,8 @@ def __init__(
         self.controller = controller
         self.entity_description = description
 
+        controller.known_objects.add((description.key, obj_id))
```

Comment:
description keys are considered unique per platform, however, this line is in your general base entity (used by all other platforms). It would be easy to cause collisions in the future.

Reply:
Sure. I want them to be enough descriptive per functionality so shouldn't be an issue.